Gemfile.lock

```diff
@@ -278,11 +331,10 @@ GEM
     websocket-extensions (0.1.5)
     xpath (3.2.0)
       nokogiri (~> 1.8)
-    zeitwerk (2.5.3)
+    zeitwerk (2.5.4)
 
 PLATFORMS
-  arm64-darwin-21
-  ruby
```

Comment:
We probably want to keep these (at least `ruby` for Docker)

Reply:
Thanks!